src/utils/RPGUtil.ts

```diff
@@ -40,7 +41,7 @@ export default class RPGUtil {
 
     return {
       name: backpack.id,
-      capacity: backpack.capacity,
+      capacity: backpack?.capacity ?? 0,
```

Comment:
Why use optional chaining on `backpack` on the capacity property but not on the name property?

Reply:
capacity is optional, but the name is not.